bin/run-scripted.sh

```diff
@@ -3,7 +3,7 @@ set -eux
 
 if [[ $(git diff master --name-only | grep sbt-metals) ]]; then
   echo "Diff detected in sbt-metals, running scripted tests"
-  sbt ^sbt-metals/scripted
+  sbt "+sbt-metals/scripted"
```

Comment:
We were doing it correctly but having `+test` for some things and `^scripted` for other things is unnecessary because we can use `+` for everything. I did this change so that `sbt ci-release` works out-of-the-box.